Unity/opentok/holo/holo/ViewController.mm

```diff
@@ -629,9 +632,11 @@ - (void)publisher:(nonnull OTPublisherKit *)publisher rtcStatsReport:(nonnull NS
         stats = [NSString stringWithFormat:@"Publisher: %@", [outboundStats length] > 0 ? outboundStats : @"NA"];
         dispatch_async(dispatch_get_main_queue(), ^{
             [self->_publisherStatsLabel removeFromSuperview];
-            [self->_publisherStatsLabel setText:stats];
-            [self->_publisherStatsLabel setNumberOfLines:0];
-            [self->_localVideoView addSubview:self->_publisherStatsLabel];
+            if(self->_enableLogs){
```

Comment:
Should also include `[self->_publisherStatsLabel removeFromSuperview];` here?

Reply:
same